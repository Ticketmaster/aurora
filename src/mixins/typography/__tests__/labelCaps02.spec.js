import LABEL_CAPS_02 from "../labelCaps02";

describe("LABEL_CAPS_02", () => {
  it("should return the correct values", () => {
    expect(LABEL_CAPS_02).toMatchSnapshot();
  });
});
