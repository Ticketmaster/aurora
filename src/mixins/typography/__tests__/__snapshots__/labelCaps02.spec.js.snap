// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LABEL_CAPS_02 should return the correct values 1`] = `
Array [
  "
  ",
  "
  ",
  "
  color: ",
  [Function],
  ";
",
  ";
  ",
  "
  ",
  "font-weight: 600",
  ";
",
  ";
  ",
  "
  line-height: ",
  "1.25",
  ";
",
  ";
",
  ";
  ",
  "
  font-size: ",
  "10px",
  ";
",
  ";
  ",
  "
    text-transform: ",
  "uppercase",
  ";
  ",
  ";
",
]
`;
